CMakeLists.txt

```diff
@@ -596,16 +596,16 @@ add_definitions(${OSRM_DEFINES})
 include_directories(SYSTEM ${DEPENDENCIES_INCLUDE_DIRS})
 
 set(BOOST_BASE_LIBRARIES
-   ${Boost_DATE_TIME_LIBRARY}
-   ${Boost_CHRONO_LIBRARY}
-   ${Boost_FILESYSTEM_LIBRARY}
-   ${Boost_IOSTREAMS_LIBRARY}
-   ${Boost_THREAD_LIBRARY}
-   ${Boost_SYSTEM_LIBRARY})
+   ${Boost_DATE_TIME_LIBRARY_RELEASE}
+   ${Boost_CHRONO_LIBRARY_RELEASE}
+   ${Boost_FILESYSTEM_LIBRARY_RELEASE}
+   ${Boost_IOSTREAMS_LIBRARY_RELEASE}
+   ${Boost_THREAD_LIBRARY_RELEASE}
+   ${Boost_SYSTEM_LIBRARY_RELEASE})
 
 set(BOOST_ENGINE_LIBRARIES
-  ${Boost_ZLIB_LIBRARY}
-  ${Boost_REGEX_LIBRARY}
+  ${Boost_ZLIB_LIBRARY_RELEASE}
+  ${Boost_REGEX_LIBRARY_RELEASE}
```

Comment:
I am not experienced with the Mason at all, but these renaming may be a problem as it does not match variable names set for the Boost via Mason

https://github.com/Project-OSRM/osrm-backend/blob/2d0b2ca98749021e3fb738b16ddd087c21f96221/CMakeLists.txt#L461-L479

Reply:
I have no experience with Mason either and missed it :( Thanks for pointing me